store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonSchemaReaderTest.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
a. No need to generate data for each testcase, can generate in beforeAll and all test case use same data. 
b. CarbonReaderTest has already covered scenarios mentioned in the below test. (both data and index file read)

**so, this test case is not required. Please remove it.**

Reply:
This test case is test for new class, like Schema. java and we should move read schema test case from CarbonReaderTest to CarbonSchemaReaderTest